### PR TITLE
write yale config

### PR DESCRIPTION
**Pull request by jacobthill**
fixes https://github.com/sul-dlss/dlme-traject/issues/43

@anarchivist Wayne couldn't remember what needed to be done with all of this legacy data but since he created a ticket for it I went ahead and wrote the config.

### CODE REVIEW

File: metadata_mapping.json [hunk 1]
```diff
@@ -250,5 +250,19 @@
...
+    "trajects": [
+      "yale_medical_config.rb"
+    ],
+    "paths": [
+      "yale/medical-manuscripts"
+    ],
+    "extension": ".csv",
+    "settings": {
+      "agg_provider": "Yale University",
+      "agg_data_provider": "Yale University",
```
Comment (anarchivist) [inline metadata_mapping.json]: `agg_data_provider` should be `"Yale University. Cushing/Whitney Medical Library"`

File: yale_medical_config.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+to_field 'cho_description', column('Abstract'), strip
+to_field 'cho_description', column('Notes'), strip
+to_field 'cho_edm_type', column('Type of resource'),
+         strip, transform(&:downcase), translation_map('not_found', 'types')
+to_field 'cho_extent', column('Physical description'), strip
+to_field 'cho_identifier', column('Accession number'), strip
+to_field 'cho_language', column('Language', split: '|'),
+         strip, transform(&:downcase), translation_map('not_found', 'languages', 'marc_languages')
+to_field 'cho_provenance', column('Associated Names'), strip
+to_field 'cho_provenance', column('Yale Collection'), strip
```
Comment (anarchivist) [inline yale_medical_config.rb]: Remove this based on other proposed change

File: yale_medical_config.rb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  provide 'reader_class_name', 'TrajectPlus::CsvReader'
+end
+
+# CHO required
+to_field 'id', normalize_prefixed_id('OrbisBibID'), strip
+to_field 'cho_title', column('Title'), strip
+
+# CHO other
+to_field 'cho_alternative', column('Variant Titles', split: '|'), strip
+to_field 'cho_coverage', column('Subject, geographic code'), strip
```
Comment (anarchivist) [inline yale_medical_config.rb]: I'd take this out - in records where this is present, it's just indicating that the volume is physically held in New Haven, Connecticut.

**Comment by jacobthill** [comment]
@anarchivist I think I removed the correct lines. Let me know if I need to change anything else

**Comment by anarchivist** [comment]
You still need to change `metadata_mapping.json`.

**Pull request merged**